.github/workflows/build-containers.yml

```
@@ -241,4 +241,12 @@ jobs:
           git add .
           git -c user.name='FlowForge Build Bot' -c user.email='noreply@flowforge.com' commit -m "Add ${{ github.ref_name }} helm chart"
           git push origin
+      - name: Resync Maintenance
+        if: ${{ endsWith(github.ref, '.0') }}
+        run: |
+          git checkout maintenance
```

I *think* you need a `cd helm` at the start here as it has checked-out the code under a path, not in the cwd.
```suggestion
          cd helm
          git checkout maintenance
```

---

Doh, yeah I just borrowed from the flowforge project